code/modules/clothing/masks/mystic.dm

```diff
@@ -1,75 +1,101 @@
+//todo:
+//ghost flickering + sprite
+//ghosts spooking near happiest mask causing an immediate flickering
+//the special stuff that happens when you get above 75 happiness
+
+#define VERYHAPPY 75
+#define HAPPIER 2
+#define UNHAPPIER 10
+
 /obj/item/clothing/mask/happy
 	name = "Happiest Mask"
-	desc = "<span class=warning>\"I'm happy! I'M HAPPY! SEE! I SAID I'M HAPPY PLEASE DON'T\"<span>"
+	desc = "<span class='sinister'>\"We're so happy AHA.\"<span>"
 	icon_state = "happiest"
 	item_state = "happiest"
 	clothing_flags = MASKINTERNALS
 	body_parts_covered = FACE
 	w_class = W_CLASS_SMALL
 	siemens_coefficient = 3.0
 	gas_transfer_coefficient = 0.90
+	var/happiness = 0
+	var/canemote
 
 /obj/item/clothing/mask/happy/equipped(M as mob, wear_mask)
 	var/mob/living/carbon/human/H = M
 	if(!istype(H))
 		return
 	if(H.wear_mask == src)
 		flick("happiest_flash", src)
-		to_chat(H, "<span class='sinister'>Your thoughts are bombarded by incessant laughter.</span>")
+		to_chat(H, "<span class='warning'>Your thoughts are bombarded by incessant laughter.</span>\n<span class='sinister'>\"Oh joy! [M.real_name]'s decided to join the party!\"</span>")
```

Comment:
Github decided to cut the line off just perfectly after <span class='sinister'> and gave no indication the line continued.